test/tests/outputs/xml/xml_distributed_out.xml.1

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
```

Comment:
Were these 4 files accidentally added? Do we need to add a line to `.gitignore` for XML files to prevent this?

Reply:
yes and yes